bundles/af-core/pom.xml

```diff
@@ -484,6 +485,7 @@
         <dependency>
             <groupId>com.adobe.aem</groupId>
             <artifactId>uber-jar</artifactId>
+            <classifier>apis</classifier>
```

Comment:
@sakshi-arora1 why have we added this, I see this was added in master/dev about 10 months ago which is not required. The build might fail due to this.